earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -25,6 +25,7 @@ sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/g
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
```

Comment:
Well, `/usr/` is supposed to have static files that don't change during the operation of software on the system. I don't think that's the case for a GEE asset root. That should be under `/var`, or, actually, `/var/opt` since GEE is under `/opt`. It could also be under `/tmp` if it only exists for the purpose of running a set of tests and is destroyed afterward.